src/OpenSage.Mods.Generals/Gui/GameOptionsUtil.cs

```diff
@@ -16,33 +18,29 @@ namespace OpenSage.Mods.Generals.Gui
 {
     public class GameOptionsUtil
```

Comment:
This class now contains most of the logic shared between `SkirmishGameOptions` and `LanGameOptions`. It was there before, but was quite a mess, with half the logic existing there and the other half being in the UI Callbacks.